<?xml version="1.0" encoding="UTF-8"?><testrun name="stringTest (1)" project="junit" tests="4" started="4" failures="0" errors="2" ignored="0">
  <testsuite name="com.deepak.junit.stringTest" time="0.065">
    <testcase name="testUppercase" classname="com.deepak.junit.stringTest" time="0.059">
      <error>java.lang.NoClassDefFoundError: org/opentest4j/AssertionFailedError&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:171)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:163)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:481)&#13;
	at com.deepak.junit.stringTest.testUppercase(stringTest.java:22)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59)&#13;
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98)&#13;
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79)&#13;
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87)&#13;
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77)&#13;
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44)&#13;
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27)&#13;
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.lang.ClassNotFoundException: org.opentest4j.AssertionFailedError&#13;
	at java.net.URLClassLoader.findClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	... 26 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="lengthBasic" classname="com.deepak.junit.stringTest" time="0.001"/>
    <testcase name="contains_Basic" classname="com.deepak.junit.stringTest" time="0.001">
      <error>java.lang.NoClassDefFoundError: org/opentest4j/AssertionFailedError&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:171)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:163)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:481)&#13;
	at com.deepak.junit.stringTest.contains_Basic(stringTest.java:33)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59)&#13;
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98)&#13;
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79)&#13;
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87)&#13;
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77)&#13;
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44)&#13;
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27)&#13;
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37)&#13;
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.lang.ClassNotFoundException: org.opentest4j.AssertionFailedError&#13;
	at java.net.URLClassLoader.findClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)&#13;
	at java.lang.ClassLoader.loadClass(Unknown Source)&#13;
	... 26 more&#13;
&#13;
</error>
    </testcase>
    <testcase name="split_Basic" classname="com.deepak.junit.stringTest" time="0.003"/>
  </testsuite>
</testrun>
